### DISCUSSION TITLE
Paid Jobs Contact Person

### DISCUSSION THREAD

**Discussion by ardha27** [davefojtik/RunPod-Fooocus-API#30] [Q&A]
Hi Dave,

I would like to get in touch with you to discuss the opportunity of working together with my team on the Runpod Fooocus API. Do you have an email or Discord contact? or please contact me on Discord "ardha27" if you interested

**Comment by davefojtik** [comment]
Hello. I am reachable on [Linkedin](https://www.linkedin.com/in/davefojtik/), `dave@qode.cc` or [Discord](https://discordapp.com/users/675731846889209882) - I sent you request there. Even though I've already turned down some jobs offered to me based on this repo in favour of my main projects in the past, I am always eager to hear about all the opportunities and amazing things that the code is used for. I would love to hear about yours!
